src/shared/lib/GenericAssayUtils.ts

```diff
@@ -0,0 +1,106 @@
+import client from 'shared/api/cbioportalClientInstance';
+import { GenericAssayTypeConstants } from 'pages/resultsView/ResultsViewPageStore';
+import { GenericAssayMeta, GenericAssayMetaFilter, GenericAssayDataFilter, GenericAssayData } from 'shared/api/generated/CBioPortalAPI';
+import { MolecularProfile } from 'shared/api/generated/CBioPortalAPI';
+import _ from 'lodash';
+import { IDataQueryFilter } from './StoreUtils';
+
+export type Treatment = {
+    'description': string;
+    'name': string;
+    'refLink': string;
+    'treatmentId': string;
+}
+
+export type TreatmentMolecularData = {
+    'geneticProfileId': string;
+    'patientId': string;
+    'sampleId': string;
+    'stableId': string;
+    'studyId': string;
+    'treatmentId': string;
+    'uniquePatientKey': string;
+    'uniqueSampleKey': string;
+    'value': string;
+}
+
```

Comment:
@alisman Since we removed the `treatment` implementation from the backend, the frontend should response to having types `Treatment` and `TreatmentMolecularData`.